FillableFieldworkAIDriver.lua

```diff
@@ -198,6 +201,11 @@ function FillableFieldworkAIDriver:areFillLevelsOk(fillLevelInfo,isWaitingForRef
 		self:debugSparse('Has no fertilizer but has seeds so keep working.')
 		allOk = true
 	end
+	-- special check if the needed fillTypes for sowing are there but a fillUnit is empty
+	if not allOk and self.vehicle.cp.settings.sowingMachineFertilizerEnabled:is(true) and not hasNoFertilizer and hasSeeds then
```

Comment:
Not sure this will always work, what would for example happen if we run out of diesel?

Reply:
I emptied the diesel and then refilled 5 liter. Once i started the cp worker he immidiatly called for refilling and stopped working. So there is no interferance.
![fsScreen_2021_09_17_13_12_41](https://user-images.githubusercontent.com/52242395/133773575-d5699e0a-1a37-4b82-b4c7-1fc81c908e89.png)